changes/add_priority_target.md

```diff
@@ -0,0 +1 @@
+priority by default to the consumableResource shesmu targets
```

Comment:
What do you think about this wording for clarity? 
`priority as a consumable resource on al Vidarr targets`

(as theoretically more things than just Shesmu could make use of the targets)